.gitignore

```diff
@@ -0,0 +1,4 @@
+# compiled binaries
```

Comment:
Looks good overall! Small nit: Should we output the build artifacts to a subdirectory such as "build/bin" or build/output/? That way we don't need to keep the gitignore updated as we add more architectures, and it's also easy to wipe out the artifacts manually.

Reply:
sounds good. Added subdirectory `build/bin`.